### ISSUE TITLE
Bug on calculating "argmax_predictions". It increases the false positive count for last label.

### ISSUE THREAD

**Issue by 12seetharaman** [allenai/allennlp#4357]
File: https://github.com/allenai/allennlp/blob/master/allennlp/training/metrics/fbeta_measure.py
Line no: 132

Prediction tensor is all zero "[0, 0, 0, 0, 0, 0]", when a record is not classified to any of the labels. 

`argmax_predictions = predictions.max(dim=-1)[1].float()`

Here, we are calculating argmax predictions by a max value index. 
In the case of all zeros tensor, we will get the index of the last label. It increases the false positive count of the last label and affects the F1 score.

```
>>> prediction =  torch.zeros(6)
>>> prediction
tensor([0., 0., 0., 0., 0., 0.])
>>> prediction.max(dim=-1)
(tensor(0.), tensor(5))
>>> prediction.max(dim=-1)[1]
tensor(5)
```
